index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Intuit Accounts</title>
+  <link rel="stylesheet" href="./styles/style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+</head>
+
+<body>
+  <header>
+    <div class="top-sign flex">
+      <div class="ts-elements">
+        <p class="ts-text">Already have an Intuit account?</p>
+        <a href="#" class="ts-text link">I forgot my User ID or Password</a>
+      </div>
+      <button class="ts-button"><i id="lock-icon" class="material-icons lock-icon">lock</i><span id="top-button-text">Sign In</span></button>
+    </div>
+  </header>
+  <main>
+    <figure class="images flex">
+      <img class="main-logo" src="img/intuit-logo1.png">
+      <ul class="other-logos flex">
+        <li>
+          <img class="logos" src="img/mint.svg"></li>
+        <li>
+          <span class="logo-text">mint</span>
+        </li>
+        <li>
+          <img class="logos" src="img/quickbooks.svg">
+        </li>
+        <li>
+          <span class="logo-text">quickbooks</span>
+        </li>
+        <li>
+          <img class="logos" src="img/turbotax.svg">
+        </li>
+        <li>
+          <span class="logo-text">turbotax</span>
+        </li>
+      </ul>
+    </figure>
+    <form class="main-form flex" action="#">
+      <section id="legend-header" class="flex">
+        <h3>Create an Intuit Account</h3>
+        <h5>One account for everything Intuit, including Mint.</h5>
+        <a href="#" class="link">Learn more</a>
+      </section>
+      <fieldset class="main-form-fields flex">
+        <label for="email">
+          Email address
+          <input type="email" name="email">
+        </label>
```

Comment:
For all the label-input tag patterns, make sure that the for="" property in the label tag, match with an id="" property used in the input.
Hint: 
![image](https://user-images.githubusercontent.com/25496166/90510803-f3e8f880-e131-11ea-8d34-56952ac0aee2.png)